src/transformers/training_args.py

```diff
@@ -989,7 +992,16 @@ class TrainingArguments:
             )
         },
     )
-
+    dataloader_prefetch_factor: int = field(
+        default=None,
+        metadata={
+            "help": (
+                "Number of batches loaded in advance by each worker. "
+                "2 means there will be a total of 2 * num_workers batches prefetched across all workers. "
+                "(default value is None)"
```

Comment:
small nit - another formatting standard of the library. We talk about args being unset if their default is None and not used in these cases.

```suggestion
                "Default is unset"
```